utils/litemap/src/map.rs

```diff
@@ -500,22 +384,11 @@ where
 impl<K, V, S> FromIterator<(K, V)> for LiteMap<K, V, S>
 where
     K: Ord,
-    S: StoreMut<K, V>,
+    S: StoreMut<K, V> + StoreFromIterable<K, V>,
```

Comment:
Nit: I think this shouldn't need `StoreMut` any more.

Reply:
Done.